knowledge/README.rst

```diff
@@ -0,0 +1,104 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================
+Knowledge Management System base
+================================
+
+This module allows you to write web pages for internal documentation.
+
```

Comment:
This module provides the base for the Knowledge applications (menu, groups, configuration panel).